.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "FieldOpt/Optimization/experimental_constraints"]
```

Comment:
Well, we don't need to, but as of now it contains nothing and has caused nothing but confusion. We can re-add it when it actually contains something.